README.fr.md

```diff
@@ -1,35 +1,28 @@
-﻿- [English](README.md)
-- [日本語](README.ja.md)
-- [한국어](README.ko.md)
-- [Bahasa Indonesia](README.in.md)
-- [Español](README.es.md)
-- [简体中文](README.zh.md)
-- [français](README.fr.md)
-- [العربية](README.ar.md)
+- [English](README.md)
 
-# Projet-allons-se-faire-arrêter
+# Projet faisons-nous-arrêter
 
-Au Japon, une collégienne a été arrêtée par la police pour avoir mis un lien vers un site Web avec une boucle d'alerte infini en JavaScript, comme ceci:
+Au Japon, une collégienne s'est faite arrêtée par la police pour avoir partagé un lien vers un site Web contenant une boucle d'alerte infinie en JavaScript, comme ceci :
```

Comment:
Maybe "une boucle infinie d'alertes" sounds better? Or maybe to be 100% clear, "une boucle infinie produisant des alertes" (an infinite loop producing alerts) although this is a bit different to the way it is worded in English.

At least I feel like "infinie" should be right after "boucle" as it is more relevant to that word than it is to "alertes"

Reply:
"une boucle infinie d'alertes" sound better